commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/dto/packet/PacketDto.java

```diff
@@ -22,6 +22,7 @@
     private Map<String, Document> documents;
     private List<Map<String, String>> audits;
     private Map<String, BiometricRecord> biometrics;
+    private int iteration;
```

Comment:
Dont use primitive types. Use wrapper class Integer instead. Because it may not be present in request that time value should be assigned to null. If u use primitive types it will be assigned to default value.

Reply:
Modified to Wrapper class